chunky/src/java/se/llbit/chunky/ui/UILogReceiver.java

```diff
@@ -34,31 +34,36 @@
 public class UILogReceiver extends Receiver {
 
   private ChunkyErrorDialog errorDialog = null;
+  private ChunkyErrorDialog warningDialog = null;
 
   @Override public void logEvent(Level level, final String message) {
+    Platform.runLater(() -> {
+      createOrGetDialogContainer(level).addMessageAndShow(message);
+    });
+  }
+
+  private ChunkyErrorDialog createOrGetDialogContainer(Level level) {
     switch (level) {
       case INFO:
       case WARNING:
-        Platform.runLater(() -> {
-          Alert warning = Dialogs.createAlert(Alert.AlertType.WARNING);
-          warning.setContentText(message);
-          warning.show();
-        });
-        break;
+        if (warningDialog == null) {
+          try { 
+            warningDialog = new ChunkyErrorDialog(level); 
```

Comment:
:bug: This will result in the warning dialog being called "Info Summary" if the level of the first message is INFO.